app/Http/Controllers/Api/V1/CurrikiGo/ExtractXAPIJSONController.php

```diff
@@ -187,10 +191,37 @@ public function runJob(ActivityRepositoryInterface $activityRepository, LRSState
                     $insertData['page'] = $insertData['object_name'];
                     $insertData['page_completed'] = $insertData['verb'] === 'completed' ? true : false;
                 }
-                
+              
                 $inserted = $lrsStatementsRepository->create($insertData);
+               
                 if ($inserted) {
                     \Log::info(date('Y-m-d h:i:s') . ' - XAPI statement with id: ' . $row->id . ' processed');
+                    //Capturing the custom verb "summary-curriki" for submit event with full summary rdbms.. 
+                    if($verb === 'summary-curriki' && !empty($interactionSummaryGlobal)){
+                        if (isset($interactionSummaryGlobal['response']) && !empty($interactionSummaryGlobal['response'])) { 
+                            $responseObject = (is_array(json_decode($interactionSummaryGlobal['response'],true)) ? json_decode($interactionSummaryGlobal['response'],true) : false);
+                            if(!empty($responseObject)){
+                                foreach($responseObject as $response){
```

Comment:
Please properly indent your code for if/else, and foreach blocks.
And add a space after comma in the parameter list.

Use PSR-12 standard.